### PR TITLE
UIP-1639 Update formatting guidelines WRT dartfmt and trailing commas

### PR DESCRIPTION
**Pull request by greglittlefield-wf**
## Ultimate problem:
Formatting guidelines were outdated when it came to trailing commas. (#25)

## How it was fixed:
Update them.

## Testing suggestions:
* Verify that the new guidelines are correct and makes sense.

## Potential areas of regression:
* README formatting

---

> __FYA:__ @greglittlefield-wf @aaronlademann-wf @jacehensley-wf @clairesarsam-wf @joelleibow-wf

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 0

File: README.md [hunk 1]
```diff
@@ -495,17 +495,50 @@ as shown in the examples above.
 
 ## Component Formatting
 
-> NOTE: At this time, we __do not recommend using [`dartfmt`](https://github.com/dart-lang/dart_style)__, as it 
-> _greatly_ decreases the readability of components built using 
-> [OverReact's fluent-style](#fluent-style-component-consumption).
-> 
-> _We have a long-term goal of writing our own formatter that extends from `dartfmt` to make it possible for our 
-> fluent-style to remain readable even after the formatter executes._
+#### dart_style
```
Comment (aaronlademann-wf) [inline README.md]: #nit shouldn't this be an `h3`?

Comment (greglittlefield-wf) [inline README.md]: Idk, I made it an h4 so it'd look less important 😄 . Didn't want to take away from the meat of this section: the formatting guidelines.

I can change it to be more consistent, though

Comment (aaronlademann-wf) [inline README.md]: @greglittlefield-wf I'd say - to make it look less important - just use bold text instead of a heading?

Comment (greglittlefield-wf) [inline README.md]: Addressed in https://github.com/Workiva/over_react/pull/37/commits/2fb8625700563667727b6e27e269ab6787335359

**Comment by aaronlademann-wf** [comment]
+1

**Comment by jacehensley-wf** [comment]
+1

**Comment by leviwith-wf** [comment]
QA +10
- readme formatting looks fine
- New guidelines read well

merging.

**Pull request merged**